modules/ocf_ldap/manifests/init.pp

```diff
@@ -5,6 +5,8 @@
   # ldif files in /etc/ldap/slapd.d, slapd is the ldap server
   package { ['slapd', 'ocf-ldap-overlay', 'libarchive-zip-perl']:; }
   service { 'slapd':
+    ensure    => 'running',
```

Comment:
Generally this isn't great to have in puppet since it leads towards a pattern where you need puppet to run on boot or something to set up a system. We don't have that, but we've specifically avoided setting this on almost all services in puppet since systemd should handle this. Did `slapd` crash or something that made this necessary?

Reply:
It wasn't running on dev-firestorm, I think. I guess `enable => true` should be sufficient?